File-Formats/DocIO/word-to-pdf.md

```diff
@@ -1692,6 +1692,514 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
+            {
+               pdfDocument.Save(outputStream);
+            }
+         }
+      }
+   }
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+//Loads an existing Word document
+using (WordDocument wordDocument = new WordDocument("Template.docx", Syncfusion.DocIO.FormatType.Docx))
+{
+   //Initialize the default fallback fonts collection
+   wordDocument.FontSettings.FallbackFonts.InitializeDefault();
+   FallbackFonts fallbackFonts = wordDocument.FontSettings.FallbackFonts;
+   foreach (FallbackFont fallbackFont in fallbackFonts) 
+   {
+      //Customize a default fallback font name as "David" for the Hebrew script
+      if (fallbackFont.ScriptType == ScriptType.Hebrew)
+         fallbackFont.FontNames = "David";
+      //Customize a default fallback font name as "Microsoft Sans Serif" for the Thai script
+      else if (fallbackFont.ScriptType == ScriptType.Thai)
+         fallbackFont.FontNames = "Microsoft Sans Serif";
+   }
+   //Instantiation of DocToPDFConverter for Word to PDF conversion
+   using (DocToPDFConverter converter = new DocToPDFConverter())
+   {
+      //Converts Word document into PDF document.
+      using (PdfDocument pdfDocument = converter.ConvertToPDF(wordDocument))
+      {
+         //Saves the PDF file to file system.
+         pdfDocument.Save("WordToPDF.pdf");
+      }
+   }
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+'Loads an existing Word document
+Using wordDocument As WordDocument = New WordDocument("Template.docx", FormatType.Docx)
+    'Initialize the default fallback fonts collection
+    wordDocument.FontSettings.FallbackFonts.InitializeDefault()
+	Dim fallbackFonts As FallbackFonts = wordDocument.FontSettings.FallbackFonts
+	foreach (FallbackFont fallbackFont in fallbackFonts) 
+    {
+      'Customize a default fallback font name as "David" for the Hebrew script
+      if (fallbackFont.ScriptType == ScriptType.Hebrew)
+         fallbackFont.FontNames = "David";
+      'Customize a default fallback font name as "Microsoft Sans Serif" for the Thai script
+      else if (fallbackFont.ScriptType == ScriptType.Thai)
+         fallbackFont.FontNames = "Microsoft Sans Serif";
+    }
+    'Instantiation of DocToPDFConverter for Word to PDF conversion
+    Using converter As New DocToPDFConverter()
+        'Converts Word document into PDF document
+        Using pdfDocument As PdfDocument = converter.ConvertToPDF(wordDocument)
+            'Saves the PDF file to file system
+            pdfDocument.Save("WordToPDF.pdf")
+        End Using
+    End Using
+End Using
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Use-alternate-font-without-installing).
+
+### Supported script types
+
+The following table illustrates the supported script types by .Net Word library (DocIO) in Word to PDF conversion.
```

Comment:
.NET Word library (DocIO)

Reply:
Modified the changes to .NET for Image and presentation controls